src/oss/confluent.sh

```diff
@@ -911,7 +911,26 @@ list_command() {
     fi
 }
 
+validate_java_version() {
+    local target_service=${1}
+    if [[ $target_service = "zookeeper" || target_service = "kafka" ]]; then
+        return
+    fi
+    local java_version=$(java -version 2>&1 |  head -n 1 | awk -F '"' '{print $2}' | awk -F'.' '{OFS = ""; print $1,$2}')
+    # Warn users if it is not Java7 or Java8, or if we can't figure it out.
+    if [[ $java_version -ne 17 && $java_version -ne 18 ]]; then
+        echo "WARNING: Java version 1.8 is recommended:"
+        echo "  See https://docs.confluent.io/current/installation/versions-interoperability.html for detailed version and interoperability information"
+    fi
+    # Fail fast if the version is definitely Java9
+    if [[ $java_version = 19 || $java_version = 110]]; then
```

Comment:
I think Java 9 will return "90" with your command, and Java 10 will return "100". We can just do `$java_version -gt 18` if you are not sure of the output of the version command.